.travis/preinstall

```diff
@@ -1,18 +1,32 @@
 #!/bin/bash
 # SPDX-License-Identifier: MIT
 
-set -ex
+# Install package specified by user in LSR_EXTRA_PACKAGES. Executed by Travis
+# during before_install phase.
+#
+# LSR_EXTRA_PACKAGES, set by user in .travis/config.sh, is a space separated
+# list of packages to be installed on the Travis build environment (Ubuntu).
 
-SCRIPTDIR=$(dirname $0)
-CONFIG=${SCRIPTDIR}/config.sh
+set -e
 
-if [[ -f ${CONFIG} ]]; then
-  . ${CONFIG}
+SCRIPTDIR=$(readlink -f $(dirname $0))
+
+# Include library.
+. ${SCRIPTDIR}/utils.sh
+
+# Add python3-selinux package (it turned out that in some circumstances, i.e
```

Comment:
AFAIU this is because molecule is using the copy or file Ansible modules on a selinux enabled system to setup the container. It has nothing to do with the tests itself but with setting up the test environment.

Reply:
Thank You for noting this, I will mention it in the comment.